### PR TITLE
Add php versions support

### PR DESCRIPTION
**Pull request by stanstst**

### CODE REVIEW

File: tests/tests/Model/PaymentTest.php [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Model_PaymentTest extends Testcase_Purchases {
 			'name'			=> 'TEST HOLDER',
 			'number'		=> '4242424242424242',
 			'expiryMonth'	=> '10',
-			'expiryYear'	=> '19',
+			'expiryYear'	=> '21',
```
Comment (RusiPapazov) [inline tests/tests/Model/PaymentTest.php]: Funny how we have to keep this updated. Can we just add 2050 as a date and not ever touch it again?

File: .travis.yml [hunk 2]
```diff
@@ -19,7 +28,6 @@ cache:
   directories:
   - $HOME/.composer/cache
 
-env:
```
Comment (hkdobrev) [inline .travis.yml]: I think this `env:` line shouldn't be deleted, but the two `env` sections, should be merged together with the new one above.

Comment (tumbalev) [reply .travis.yml]: Yes, sure. I've merged them. Thanks!

**Comment by tumbalev** [comment]
There is a problem with the lowest deps on `thephpleague/omnipay:v3.0.0`. It requires "php-http/guzzle6-adapter": "^1.1" which have a problem with instantiation of the client (fixed in v2 - https://github.com/php-http/guzzle6-adapter/compare/v2.0.0...master#diff-ff73e042e738204c6da009e2ed19f783R31). As a result it seems like we can't do nothing, except removing the lowest deps in Travis.

**Comment by tumbalev** [comment]
> * Find the exact omnipay version which is good for us and require it as a minimum in the constraint (e.g. `^3.0.1`)
> * Use the locked version which is working for us as a minimum in the constraint (`^3.0.2`)

@hkdobrev I tried the mentioned solution, but it doesn't work either. I got the v1.1 of the `php-http/guzzle6-adapter`. Because of that, I got the following exception:
```
Message was: "Too few arguments to function Http\Adapter\Guzzle6\Client::buildClient(), 0 passed in openbuildings/purchases/vendor/php-http/guzzle6-adapter/src/Client.php on line 31 and exactly 1 expected" at
openbuildings/purchases/vendor/php-http/guzzle6-adapter/src/Client.php:76
openbuildings/purchases/vendor/php-http/guzzle6-adapter/src/Client.php:31
```

Any other suggestion?

**Comment by hkdobrev** [comment]
This means we need to add the locked version of the guzzle library as a minimum requirement in the composer.json of the purchases package. This way we would prevent this library to be possible to be installed in a project with a lower version of guzzle.

**Comment by tumbalev** [comment]
> This means we need to add the locked version of the guzzle library as a minimum requirement in the composer.json of the purchases package. This way we would prevent this library to be possible to be installed in a project with a lower version of guzzle.

@hkdobrev I need to lock the `omnipay` and `php-http/httplug` (it is a guzzle's dependency) in order to fix the issue. Is that ok?

**Pull request merged**